src/app/(dashboard)/a/settings/component/SMTPSettings-card.tsx

```diff
@@ -0,0 +1,104 @@
+'use client'
+import { Button } from "@/components/ui/button"
+import { Card } from "@/components/ui/card"
+import { Label } from "@/components/ui/label"
+import { toast } from "@/components/ui/use-toast"
+import Link from "next/link"
+import { useForm } from "react-hook-form"
+import { IoSettingsSharp } from "react-icons/io5"
+
+export default function SettingsCard() {
+  const {
+    register,
+    handleSubmit
+  } = useForm({})
+
+  const onSubmit = async (formData: any) => {
+    try {
+      formData.port = parseInt(formData.port, 10)
+      formData.secure = parseBoolean(formData.secure)
+      const response = await fetch('/api/settings/smtp', {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify(formData),
+      });
+
+      if (response.ok) {
+        toast({
+          title: 'Sending data to backend',
```

Comment:
Put a meaningful message after saving the settings. Also, are we redirecting the user to somewhere else after saving the settings?

Reply:
Done change the message. No, currently we are not redirecting to another page.